docs/builtins/Capabilities/require-capability.md

```diff
@@ -1,30 +1,45 @@
 ## require-capability
-The `require-capability` function specifies and tests for the existing grant of a specified `CAPABILITY`, failing if it is not found in the environment.
+
+Use `require-capability` to require a specific `CAPABILITY` to be granted before allowing the current body of code to be executed.
+If the required capability isn't found in the environment, the code fails to execute.
+
+By convention, capabilities are defined using all uppercase letters.
 
 ### Basic syntax
 
-To specify and test for the existing grant of a `CAPABILITY`, use the following syntax:
+To require a specific `CAPABILITY` to be granted, use the following syntax:
```

Comment:
@lsgunnlsgunn , the cap has to be granted. Is this clear here?

Reply:
Maybe not (I'm still trying to wrap my head around capabilities tbh). I think what we are trying to communicate is:
You must define, compose, or install a capability (a guard/predicate function that says what condition must be met to grant an elevated privilege). If the condition is true, the capability is granted/acquired/installed and thereafter you can use with-capability or require-capability to control when/where the elevated permission is used.
Is that somewhat accurate?